alfasim_sdk/cli.py

```diff
@@ -108,6 +140,21 @@ def _compile(plugin_dir):
 @destination_option(help="A path to where the output package should be created.")
 @click.option("--package-name", prompt="-- Package Name", help="Name of the package")
 def package(ctx, plugin_dir, package_name, dst):
+    """
+    Creates a new ``<package-name>.hmplugin`` file containing all the necessary files.
+
+    This command will first invoke the compile command to generate the shared library, and after that, the plugin
+    package will be generated  with all the content available from the directory assets and artifacts.
+
+    Per default, the package command will assume that the plugin project is the current directory and the generated file
+    will be placed also in the current directory.
+
+    In order to change that, it's possible to pass the following options:
+
+     -- plugin-dir with the path to where the plugin directory, where configuration and the shared library is located.
```

Comment:
These two items are not being listed as items in the rendered manual:

![image](https://user-images.githubusercontent.com/3778802/67237184-26966780-f421-11e9-832e-6b2f6e296f3c.png)

Also, reprhase: _with the path to the plugin directory_ or _with the path to where the plugin directory is_

Reply:
Changed to just:
```
In order to change that, it's possible to use the options ``plugin-dir`` and ``dst``